packages/resizin/package.json

```diff
@@ -34,6 +34,6 @@
     "isomorphic-fetch": "3.0.0",
     "lodash": "^4.17.20",
     "promise-polyfill": "^8.1.3",
-    "uuid": "^3.3.3"
+    "uuid": "^8.3.2"
```

Comment:
Recently I have read some articles that this UUID library is not so good for performance..... as an alternative is highly recommended to use NANOID - https://openbase.com/js/nanoid but I have never tried it before :)

Reply:
maybe in that newest version the uuid is fixed

Reply:
Good point, will check it out 👍